docs/resources/kafka_acl.md

```diff
@@ -7,7 +7,7 @@ The Resource Kafka ACL resource allows the creation and management of ACLs for a
 ```hcl
 resource "aiven_kafka_acl" "mytestacl" {
     project = "${aiven_project.myproject.project}"
-    service_name = "${aiven_service.myservice.service_name}"
```

Comment:
Let's use `aiven_kafka.myservice.service_name` instead of  `"${aiven_kafka.myservice.service_name}"` and the same for `aiven_project.myproject.project`